README.md

```diff
@@ -82,6 +83,7 @@ const limitd = new Limitd({
 - `unlimited` (boolean = false): unlimited requests (skip take).
 - `skip_n_calls` (number): take will go to redis every `n` calls instead of going in every take.
 - `elevated_limits` (object): elevated limits configuration that kicks in when the bucket is empty. Please refer to the [ERL section](#ERL-Elevated-Rate-Limits) for more details.
+- `fixed_window` (boolean = false): refill at specified interval instead of granular.
```

Comment:
Quoting from below in the README:

> If the param is not provided, it is interpreted as if the fixed_window parameter is true

Should this change the default value to `true`?

```suggestion
- `fixed_window` (boolean = true): refill at specified interval instead of granular.
```

Reply:
that is the bucket config, not the override

Reply:
oops my bad